src/DependencyInjection/SymfonycastsSassExtension.php

```diff
@@ -77,6 +78,12 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                     ->defaultValue(['%kernel.project_dir%/assets/styles/app.scss'])
                 ->end()
+                ->arrayNode('load_paths')
+                    ->info('Additional paths to load')
+                    ->scalarPrototype()
+                        ->end()
+                    ->defaultValue([])
```

Comment:
this is useless. prototyped nodes always have a default value automatically.

Reply:
Fixed